spread.yaml

```diff
@@ -596,12 +604,12 @@ prepare: |
                 apt-get update >& "$tf" || ( cat "$tf"; exit 1 )
                 apt-get install -y xdelta3 curl eatmydata >& "$tf" || ( cat "$tf"; exit 1 )
                 ;;
-            fedora-*)
-                dnf install --refresh -y xdelta curl &> "$tf" || (cat "$tf"; exit 1)
-                ;;
-            amazon-*|centos-*)
+            amazon-*|centos-7-*)
                 yum install -y xdelta curl &> "$tf" || (cat "$tf"; exit 1)
                 ;;
+            fedora-*|centos-*)
```

Comment:
Won't this run again on `centos-7` machines? This will fail unless you preinstall `dnf` on the CentOS 7 machines...

Reply:
No, we use `<sys>-<version>-<arch>` naming pattern, so CentOS systems are named `centos-7-64` (well, there's no 32 variant TBH), so it hits the case clause right above. Anything below 7 isn't really supported, so there's no spread systems defined for that.